src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ResendToGapsAboutToSubmitHandler.java

```diff
@@ -21,26 +23,22 @@
 import uk.gov.hmcts.reform.sscs.robotics.RoboticsValidationException;
 import uk.gov.hmcts.reform.sscs.robotics.RoboticsWrapper;
 
-@Component
 @Slf4j
+@Component
+@RequiredArgsConstructor
 public class ResendToGapsAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
 
-    private PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse;
+    @Value("${feature.gaps-switchover.enabled:false}")
```

Comment:
Does the : notation work here?

Reply:
should do.. how do you guys do it?

Reply:
It's usually just in the application.yaml file

Reply:
okay thanks I'll change it